### PR TITLE
fix: grammar error

### PR DESCRIPTION
**Pull request by yoyo314**

### CODE REVIEW

File: docs/integration/go/bookstack.md [hunk 1]
```diff
@@ -39,6 +39,5 @@ Make sure the `Redirect URL` is the same as the `redirectUrl` in the file `oauth
 
 **Now that you've done all the configuration for Casdoor!**
 
-You can go back to your BookStack and experience using Casdoor 
-for login authentication once the BookStack has been successfully deployed.
+ go back to your BookStack and experience using casdoor for login authentication once the BookStack has been successfully deployed.
```
Comment (leo220yuyaodog) [inline docs/integration/go/bookstack.md:42]: Plz restore the changes I demonstrated yesterday.

File: docs/overview.mdx [hunk 2]
```diff
@@ -23,21 +23,21 @@ Casdoor serves both the web UI and the login requests from the application users
 
 1. Front-end and back-end separate architecture, developed by Golang, Casdoor supports high concurrency, provides web-based managing UI and supports multiple languages(Chinese, English).
 
-2. Casdoor supports Github, Google, QQ, WeChat third-party applications login, and support the extension of third-party login with plugins.
+2. Casdoor supports third-party applications including Github, Google, QQ, WeChat log in , and supports the extension of third-party login with plugins.
 
 3. With [Casbin](https://casbin.org/) based authorization management, Casdoor supports ACL, RBAC, ABAC, RESTful accessing control models.
 
-4. Phone verification code, email verification code and forget password features.
+4. Phone verification code, email verification code and password retrival functions.
```
Comment (Selflocking) [inline docs/overview.mdx:30]: I didn't find the word: `retrival`, do you mean: `retrieval`?

Comment (yoyo314) [reply docs/overview.mdx:30]: Yes, a spelling mistake, thanks for correcting!

File: docs/overview.mdx [hunk 3]
```diff
@@ -23,28 +23,28 @@ Casdoor serves both the web UI and the login requests from the application users
 
 1. Front-end and back-end separate architecture, developed by Golang, Casdoor supports high concurrency, provides web-based managing UI and supports multiple languages(Chinese, English).
 
-2. Casdoor supports Github, Google, QQ, WeChat third-party applications login, and support the extension of third-party login with plugins.
+2. Casdoor supports third-party applications login, such as Github, Google, QQ, wechat, etc.,and supports the extension of third-party login with plugins.
```
Comment (Selflocking) [inline docs/overview.mdx:26]:
```suggestion
2. Casdoor supports third-party applications login, such as GitHub, Google, QQ, WeChat, etc.,and supports the extension of third-party login with plugins.
```

**Review by leo220yuyaodog — APPROVED** [review]

File: docs/integration/java/RuoYi.md [hunk 4]
```diff
@@ -75,33 +75,31 @@ public R<?> callback(@RequestBody CodeBody code) {//we should define a CodeBody
...
+RuoYi back-end has perfect login and registration function. We just need to change a little, so it is very convenient.
+## Step 6. Detailed steps
+1. Deploy and configure casdoor. We must take care of the organization's password type which should choose bcrypt because RuoYi-Cloud's password type is bcrypt.
+2. We should use casdoor syncers to copy database users to your casdoor organization. This step can make the original account import to casdoor.
 3. After we deployed casdoor, we should change front-end. We should close RuoYi check code 
 ![checkcode Switch](/img/RuoYi-Cloud_loginSwitch.png)
 
  Note that RuoYi-Cloud checkcode needs close in nacos again.
- Note that RuoYi-Cloud open register function need change sys.account.registerUser to true. 
+ Note that if you want to register RuoYi-Cloud, you need to ensure the authenticity of your account and user name.
```
Comment (Selflocking) [inline docs/integration/java/RuoYi.md:96]: I don't think it's a grammar error.
`sys.account.registerUser` is a option, open registration function requires this option to be turned on.

**Review by Selflocking — CHANGES_REQUESTED** [review]

File: docs/integration/java/RuoYi.md (hunk 4 above)
Comment (yoyo314) [reply docs/integration/java/RuoYi.md:96]: OK, I got it! I'll revise it in my next commit.

File: docs/integration/java/Pulsar-manager.md [hunk 5]
```diff
@@ -20,21 +20,21 @@ After a successful deployment, you need to ensure:
 - Open your favorite browser and visit **http://localhost:7001**, you will see the login page of Casdoor.
 - Input `admin` and `123` to test login functionality is working fine.
 
-Then you can quickly implement a casdoor based login page in your own app with the following steps.
+Then you can quickly implement a casdoor-based login page in your own app with the following steps.
 
 ## Step2. Configure Casdoor 
-Configure casdoor can refer to [casdoor](https://door.casdoor.com/login)(Configure casdoor's browser better not use one browser with your develop browser).
+Configure casdoor can refer to [casdoor](https://door.casdoor.com/login)(Configure casdoor's browser better not use one browser as your develop browser).
```
Comment (Selflocking) [inline docs/integration/java/Pulsar-manager.md:26]: @jakiuncle plz review this.
do you mean:
> It is better not to use the same browser to configure and develop Casdoor

Comment (jakiuncle) [reply docs/integration/java/Pulsar-manager.md:26]: yes

File: docs/integration/java/RuoYi.md (hunk 4 above)
Comment (yoyo314) [reply docs/integration/java/RuoYi.md:96]: OK, I got it! I'll revise it in my next submission.

File: docs/integration/java/Pulsar-manager.md (hunk 5 above)
Comment (yoyo314) [reply docs/integration/java/Pulsar-manager.md:26]: In fact, I didn't understand what the author was trying to say.

File: docs/integration/go/gitea.md [hunk 6]
```diff
@@ -39,15 +39,15 @@ Press the "Add Authentication Source" Button, and fill in the form like this.
...
+Please input a name for this authentication source and **remember this name**. This name will be used for the callback_url in the next step.
 
-Plese choose the `OpenID Connect` Oauth2 Provider.
+Please choose the `OpenID Connect` Oauth2 Provider.
 
-Fill in the client ID and client secret remembered in previous step.
+Fill in the client ID and client secret remembered in the previous step.
 
-Fill in the openid connect auto discovery url, which is supposed to be `<your endpoint of casdoor>/.well-known/openid-configuration`.
+Fill in the open id connect auto discovery url, which is supposed to be `<your endpoint of casdoor>/.well-known/openid-configuration`.
```
Comment (jakiuncle) [inline docs/integration/go/gitea.md:50]: Maybe is OpenID?

Comment (yoyo314) [reply docs/integration/go/gitea.md:50]: ok, thx for correcting!

**Review by jakiuncle — APPROVED** [review]

File: docs/integration/go/gitea.md [hunk 7]
```diff
@@ -39,15 +39,15 @@ Press the "Add Authentication Source" Button, and fill in the form like this.
...
+Please choose the authentication type as "oauth2".
 
-Please input a name for this authentication source and **remember this name**. This name will be used for the callback_url in next step.
+Please input a name for this authentication source and **remember this name**. This name will be used for the callback_url in the next step.
 
-Plese choose the `OpenID Connect` Oauth2 Provider.
+Please choose the `OpenID Connect` Oauth2 Provider.
 
-Fill in the client ID and client secret remembered in previous step.
+Fill in the client ID and client secret remembered in the previous step.
```
Comment (leo220yuyaodog) [inline docs/integration/go/gitea.md:48]:
```suggestion
Fill in the **Client ID** and **Client Secret** remembered in the previous step.
```

File: docs/integration/go/gitea.md (hunk 6 above)
Comment (leo220yuyaodog) [inline docs/integration/go/gitea.md]:
```suggestion
Fill in the **OpenID Connect Auto Discovery URL**, which is supposed to be `<your endpoint of casdoor>/.well-known/openid-configuration`.
```

**Review by leo220yuyaodog — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
Replaced by: https://github.com/casdoor/casdoor-website/pull/358

**Pull request closed without merging**